packages/zoe/test/swingsetTests/zoe-metering/bootstrap.js

```diff
@@ -9,9 +9,12 @@ import testBuiltinsBundle from './bundle-testBuiltins';
 
 function build(E, log) {
   const obj0 = {
-    async bootstrap(argv, vats) {
-      const zoe = await E(vats.zoe).getZoe();
-
+    async bootstrap(argv, vats, devices) {
+      const zoeBuilder = await E(vats.zoe).getZoe();
+      const vatAdminSvc = await E(vats.vatAdmin).createVatAdminService(
+        devices.vatAdmin,
+      );
+      const zoe = await E(zoeBuilder).buildZoeService(vatAdminSvc);
```

Comment:
I'd look for a way to flip these around and have a single `zoe = await E(vats.zoe).makeZoe(vatAdminSvc)` call

Also, is the `await` important? I imagine there might be sequencing reasons, but if everything else here uses `E(zoe)` anyways, then a promise for Zoe is just as good as a Presence.